python/GafferTest/TweakPlugTest.py

```diff
@@ -339,6 +339,28 @@ def testTweakModes( self ) :
 		self.assertFalse( tweaks.applyTweaks( parameters ) )
 		self.assertNotIn( "f", parameters )
 
+	def testTweakInternedString( self ) :
+
+		data = IECore.CompoundData(
+			{
+				"a" : IECore.InternedStringData( "testInternedString" ),
+				"b" : "testString",
+			}
+		)
+		self.assertEqual( data["a"], IECore.InternedStringData( "testInternedString" ) )
+		self.assertEqual( data["b"], IECore.StringData( "testString" ) )
+
+		tweak1 = Gaffer.TweakPlug( "a", "stringValue" )
+		tweak2 = Gaffer.TweakPlug( "b", IECore.InternedString( "internedStringValue" ) )
```

Comment:
What is this intended to be testing? Since we don't have an InternedStringPlug type, this creates a TweakPlug where the value is held by a StringPlug, so I don't think it's providing any test coverage for the "tweak string using internedstring" code.

Reply:
I didn't know there isn't an `InternedStringPlug`, so this and accompanying code with commments below was an effort to provide symmetry between `InternedString` and `string` data. I'll pull out the parts related to the non-existent `InternedStringPlug`.